modules/statics/src/coins.ts

```diff
@@ -17673,6 +17722,14 @@ export const coins = CoinMap.fromCoins([
     '0x143e1dae4f018ff86051a01d44a1b49b13704056',
     UnderlyingAsset['tarbeth:link']
   ),
+  tarbethErc20(
+    '64477af2-65cb-44d2-a3fd-aed07cfe6bfd',
+    'tarbeth:xsgd',
+    'XSGD',
+    6,
```

Comment:
Seems like in the explorer there is no decimal value mentioned. Also can u verify the contract address with Product team?